src/mtype.h

```diff
@@ -346,6 +346,9 @@ struct mtype {
         ::weakpoints weakpoints;
         weakpoint_families families;
 
+        // Traps avoided by this monster
+        std::vector<std::string> trap_avoids;
```

Comment:
Little more convenient for what you're using it for.
```suggestion
        std::set<std::string> trap_avoids;
```

Reply:
Resolved